seacatauth/openidconnect/handler/public_keys.py

```diff
@@ -23,11 +23,14 @@ def __init__(self, app, oidc_svc):
 		self.OpenIdConnectService = oidc_svc
 
 		web_app = app.WebContainer.WebApp
-		web_app.router.add_get("/openidconnect/public_keys", self.public_keys)
+		# It is a convention to expose the JWKS at /.well-known/jwks.json
+		web_app.router.add_get("/.well-known/jwks.json", self.public_keys)
```

Comment:
do not add this. nginx should proxy the well-known urls.
this should be also stated in docstring.

Reply:
update: /.well-known subpath is okay. nginx should add a root-level proxy if needed